analytics/terraform/superset-on-eks/README.md

```diff
@@ -0,0 +1,54 @@
+# Apache Superset on EKS
```

Comment:
You don't need all this content in the readme. Checkout the sample readme from other blueprints 
https://github.com/awslabs/data-on-eks/blob/main/analytics/terraform/spark-k8s-operator/README.md